classes/class-nets-easy-email.php

```diff
@@ -32,9 +32,10 @@ public function __construct() {
 		 * @return void
 		 */
 		public function email_extra_information( $order, $sent_to_admin, $plain_text = false ) {
-			$settings     = get_option( 'woocommerce_dibs_easy_settings' );
-			$order_id     = $order->get_id();
-			$gateway_used = get_post_meta( $order_id, '_payment_method', true );
+			$settings                = get_option( 'woocommerce_dibs_easy_settings' );
+			$email_nets_payment_data = $settings['email_nets_payment_data'] ?? 'yes';
```

Comment:
You have it set to `yes` here as a fallback, but the default is `no` for the setting. Should one of them be changed?